modules/node/src/transfer/transfer.provider.ts

```diff
@@ -45,7 +45,7 @@ export class TransferMessaging extends AbstractMessagingProvider {
     return await this.transferService.resolveLinkedTransfer(pubId, paymentId, linkedHash);
   }
 
-  // TODO: types
+  // DEPRECATED
```

Comment:
can we remove?

Reply:
I don't think this was ever used externally

Reply:
Yes I would love to.

Reply:
This is a breaking change (old clients won't be able to transfer with this, but we can bump major version). That's why we should put other breaking changes into this PR.